src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -7412,6 +7417,7 @@ interface CorpConstants {
   maxProductsBase: number;
   /** The minimum decay value for morale/energy */
   minEmployeeDecay: number;
+  smartSupplyUseOptions: CorpSmartSupplyOption[];
```

Comment:
Since this is being just now added to the player facing API, we can update the name to make a little more sense. I'm not sure why `Use` is in the title but it's a weird mismatch between the type name and the variable name. smartSupplyOptions makes more sense

Reply:
will it be ok to change? if it breaks someone's script (which I doubt) - will it still be fine?
this const was not mentioned in docs before, but it was still available.

Reply:
one of the options is to add both `smartSupplyUseOptions` and `smartSupplyOptions` but mark the first one as deprecated.

Reply:
Oh, it looks like it was being actually provided in the object returned from getConstants, but just had not been added to the CorpConstants interface. It's a new addition from 2.3.0. It was never documented as being provided in the API until you just added documentation for it, I say go ahead and just change it.

Reply:
done 👍